### ISSUE TITLE
Even uber evil doesn't help with chairman

### ISSUE THREAD

**Issue by AlexCzar** [ihtasham42/progress-knight#10]
I have more than 40k in evil at this point, the Chairman is still too slow to achieve.
With 1800 in normal multipliers and 2800 in Evil ones, Chairman levelling up is still too slow, I cannot even break lvl100.
Is it broken or am I missing something?

**Comment by RedDeathNova** [comment]
You can just level up astral body and increase game speed or restart with the eye a bunch then it will have so much do multipliers it will Just level up quicker

**Comment by AlexCzar** [comment]
There is no astral body level-up in the game. And I've levelled up evil as far as I could. Diminishing returns kicked in.
Evil past 50k doesn't do anything noticeable.
Restarting with the eye stopped being useful as well.

**Comment by RedDeathNova** [comment]
I use the mobile and if you just rebirth and don't go and revert every time for evil you can easily get all this XP boost to level up and I can get chairman before year like 300 and I can easily get like everything very quickly if you just rebirth so many times and just use your first life as a XP gathering source for all extra XP boost you can easily level up as fast as you want I can easily do it I don't know how maybe it's just something special or maybe it's a tactic I use but I can level up almost instantly I can speed run this game I can get every magic upgrade before I hit year 20 I only have like 20,000 evil my highest chairman level so far is 328

**Comment by AlexCzar** [comment]
I guess, the mobile version you're playing and this game are different, as you mentioned "astral body" which does not exist in this game. I can't even get Chairman 100...
Can you share a screenshot of your game with skills visible?

**Comment by RedDeathNova** [comment]
Sure thing I got it from the App Store on my iPhone so I don't know but here are some screenshots!![image](https://github.com/ihtasham42/progress-knight/assets/143959683/788d8b7a-2fde-4687-828a-f8071792a546) ![image](https://github.com/ihtasham42/progress-knight/assets/143959683/b94a2d5e-64ac-4cc8-9c91-d2f22457f1c3) ![image](https://github.com/ihtasham42/progress-knight/assets/143959683/24c4f4c8-019a-4824-b7a8-13133d8791c4)

**Comment by AlexCzar** [comment]
Yup, looks like it's a fork, not the original game.
Thanks for confirming, at least I can now investigate further.

**Comment by RedDeathNova** [comment]
Yeah I play on a iPhone so I'm guessing on a website or something but yeah I tried to help I wish you luck in your search
